docs/cell_census_schema_0.0.1.md

```diff
@@ -0,0 +1,853 @@
+
+The main goals of the Cell Census are:
+
+1. To allow dataset-independent querying of single-cell data for reproducible analysis by computational biologists via Python and R [APIs](https://github.com/chanzuckerberg/cell-census) that enable on-demand generation of AnnData and Seurat objects respectively.
+1. To enable features of CELLxGENE Discover to access versioned snapshots of its data on cross-dataset features like [Gene Expression](https://cellxgene.cziscience.com/gene-expression). 
+
+To better understand this document the reader should be familiar with the [CELLxGENE dataset schema](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md) and [SOMA](https://github.com/single-cell-data/SOMA/blob/main/abstract_specification.md).
+
+## Definitions
+
+The following terms are used throughout this document:
+
+* adata – generic variable name that refers to an [`AnnData`](https://anndata.readthedocs.io/) object.
+* CELLxGENE dataset schema – the data schema for h5ad files served by CELLxGENE Discover, for this Cell Census schema: [CELLxGENE dataset schema version is 3.0.X](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md)
+* census\_obj – the Cell Census root object, a SOMACollection.
+* Cell Census release – a Cell Census deposited in a public bucket and accessible by APIs.
+* tissue – original tissue annotation.
+* tissue\_general – high-level mapping of a tissue, e.g. “Heart” is the tissue_general of “Heart left ventricle” .
+
+## Cell Census Schema versioning
+
+The Cell Census Schema follows [Semver](https://semver.org/) for its versioning:
+
+* Major: any schema changes that make the Cell Census incompatible with public API(s) or SOMA. Examples:
+	* Column deletion in Cell Census `obs`
+	* Addition of new modality
+* Minor: schema additions that are compatible with public API(s) and SOMA. Examples:
+	* New column to Cell Census `obs` is added
+	* tissue/tissue_general mapping changes
+* Patch: schema fixes. Examples:
+	* Editorial schema changes
+
+
+Changes MUST be documented in the schema [Changelog](#changelog) at the end of this document.
+
+Cell Census data releases are versioned separately from the schema.
+
+
+## Schema
+
+### Data included
+
+All datasets included in the Cell Census MUST be of [CELLxGENE dataset schema version 3.0.X](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md). The following data constraints are imposed on top of the CELLxGENE dataset schema.
+
+#### Species
+
+Only observations (cells) with an  [`organism_ontology_term_id`](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md#organism_ontology_term_id) value of either “NCBITaxon:10090” for *Mus musculus* or “NCBITaxon:9606” for *Homo sapiens* MUST be included.
```

Comment:
I may be parsing this sentence incorrectly - but I believe as written it could be interpreted as "other species MAY be included".

Suggested edit:   The Cell Census MUST only contain cells with an organism_ontology_term_id value of <human> or <mouse>